static/src/components/builder/components/TextInput/TextInputForm.vue

```diff
@@ -60,19 +78,43 @@ import * as types from '../../store/types';
 import { mapMutations, mapState } from 'vuex';
 export default {
   name: 'TextInputForm',
+    data () {
+    return {
+      inputTypeOptions: {
+        Text: 'text',
+        Number: 'number',
+        Date: 'date',
+        Time: 'time',
+        Email: 'email',
+        URL: 'url'
+      }
+    };
+  },
   computed: {
     ...mapState({
       label: state => state.textInput.label,
       labelAdded: state => state.textInput.labelAdded,
+      type: state => state.textInput.type,
       pybanwer: state => state.textInput['pyb-answer']
     })
   },
   methods: {
     ...mapMutations({
       'updateLabel': types.MUTATE_TEXT_INPUT_LABEL,
       'updateLabelAdded': types.MUTATE_TEXT_INPUT_LABEL_ADDED,
+      'updateType': types.MUTATE_TEXT_INPUT_TYPE,
       'updatePybanswer': types.MUTATE_TEXT_INPUT_PYB_ANSWER
     })
   }
 };
 </script>
+<style scoped>
+.block-label {
```

Comment:
can we use the bootstrap classes instead, consistently with the other form inputs? E.g.:

```html
<div class="form-row">
  <div class="form-group col-md-12">
    <label>
      Input Type
      <select
        class="form-control form-control-sm dropdown-item"
        :value="type"
        @input="updateType($event.target.value)"
      >
        <option
          v-for="(v, k) in inputTypeOptions"
          :key="k"
          :value="v"
        >
          {{ k }}
        </option>
      </select>
    </label>
  </div>
</div>
```

Reply:
I was planing to keep consistency with the drop-down component? https://github.com/bloomberg/pybossa-default-theme/blob/765d93faea406b5827f9f62bda160aa76b56738a/static/src/components/builder/components/DropdownInput/DropdownForm.vue#L20

Reply:
I didn't see that before. To me it doesn't make sense to change the typography locally like this.